[2023-05-10T18:37:27.644+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=our_first_dag_v5/run_id=manual__2023-05-10T18:37:21.261338+00:00/task_id=second_task permission to 509
[2023-05-10T18:37:31.348+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=our_first_dag_v5/run_id=manual__2023-05-10T18:37:21.261338+00:00/task_id=second_task permission to 509
[2023-05-10T18:37:31.586+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v5.second_task manual__2023-05-10T18:37:21.261338+00:00 [queued]>
[2023-05-10T18:37:31.615+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v5.second_task manual__2023-05-10T18:37:21.261338+00:00 [queued]>
[2023-05-10T18:37:31.618+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 6
[2023-05-10T18:37:31.686+0000] {taskinstance.py:1350} INFO - Executing <Task(BashOperator): second_task> on 2023-05-10 18:37:21.261338+00:00
[2023-05-10T18:37:31.692+0000] {standard_task_runner.py:57} INFO - Started process 1422 to run task
[2023-05-10T18:37:31.697+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'manual__2023-05-10T18:37:21.261338+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpmsk2nr6f']
[2023-05-10T18:37:31.700+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask second_task
[2023-05-10T18:37:31.735+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:188 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-05-10T18:37:31.804+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=our_first_dag_v5/run_id=manual__2023-05-10T18:37:21.261338+00:00/task_id=second_task permission to 509
[2023-05-10T18:37:31.807+0000] {task_command.py:410} INFO - Running <TaskInstance: our_first_dag_v5.second_task manual__2023-05-10T18:37:21.261338+00:00 [running]> on host 9a2b13458d73
[2023-05-10T18:37:32.065+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v5' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2023-05-10T18:37:21.261338+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-05-10T18:37:21.261338+00:00'
[2023-05-10T18:37:32.071+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-05-10T18:37:32.079+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2023-05-10T18:37:32.119+0000] {subprocess.py:86} INFO - Output:
[2023-05-10T18:37:32.123+0000] {subprocess.py:93} INFO - hey, I am task2 and will be running after task1!
[2023-05-10T18:37:32.124+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-05-10T18:37:32.232+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20230510T183721, start_date=20230510T183731, end_date=20230510T183732
[2023-05-10T18:37:32.317+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-10T18:37:32.405+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
